pkg/util/helpers.go

```diff
@@ -28,21 +28,21 @@ import (
 var osReleasePath = "/etc/os-release"
 
 // GenerateConditionChangeEvent generates an event for condition change.
-func GenerateConditionChangeEvent(t string, status types.ConditionStatus, reason string, timestamp time.Time) types.Event {
+func GenerateConditionChangeEvent(reason string, timestamp time.Time,msg string) types.Event {
 	return types.Event{
 		Severity:  types.Info,
 		Timestamp: timestamp,
 		Reason:    reason,
-		Message:   fmt.Sprintf("Node condition %s is now: %s, reason: %s", t, status, reason),
```

Comment:
change to `fmt.Sprintf("Node condition %s is now: %s, reason: %s, msg: %s", t, status, reason, msg)`

Reply:
done.